R/geom-freqpoly.r

```diff
@@ -16,10 +16,18 @@ geom_freqpoly <- function(mapping = NULL, data = NULL,
     data = data,
     mapping = mapping,
     stat = stat,
-    geom = GeomPath,
+    geom = GeomFreqploy,
```

Comment:
why change this?

Reply:
I had to introduce a new class `GeomFreqPoly` because there were certain operations which had to be done for FreqPoly geom only and that's why I had to change from `GeomPath` to `GeomFreqpoly`. The class `GeomFreqpoly` inherits `GeomPath` so everything should be good to go I believe.

Reply:
that makes sense thanks